src/SerialInputHandler.cpp

```diff
@@ -123,52 +132,67 @@ void _handleFactoryResetCommand(char* arg, std::size_t argLength) {
 
 void _handleRmtpinCommand(char* arg, std::size_t argLength) {
   if (arg == nullptr || argLength <= 0) {
-    Serial.println("SYS|Error|Invalid argument");
+    // Get rmt pin
+    Serial.print("$SYS$|Response|RmtPin|");
+    Serial.println(Config::GetRFConfig().txPin);
     return;
   }
 
   std::uint32_t pin;
   if (sscanf(arg, "%u", &pin) != 1) {
-    Serial.println("SYS|Error|Invalid argument (not a number)");
+    Serial.println("$SYS$|Error|Invalid argument (not a number)");
     return;
   }
 
   OpenShock::CommandHandler::SetRfTxPin(pin);
 
-  Serial.println("SYS|Success|Saved config");
+  Serial.println("$SYS$|Success|Saved config");
 }
 
 void _handleAuthtokenCommand(char* arg, std::size_t argLength) {
   if (arg == nullptr || argLength <= 0) {
-    Serial.println("SYS|Error|Invalid argument");
+    Serial.println("$SYS$|Error|Invalid argument");
     return;
   }
 
   OpenShock::Config::SetBackendAuthToken(std::string(arg, argLength));
 
-  Serial.println("SYS|Success|Saved config");
+  Serial.println("$SYS$|Success|Saved config");
 }
 
 void _handleNetworksCommand(char* arg, std::size_t argLength) {
+
+
   if (arg == nullptr || argLength <= 0) {
-    Serial.println("SYS|Error|Invalid argument");
+    // Get networks
+    StaticJsonDocument<1024> outDoc;
+    JsonArray outNetworks = outDoc.to<JsonArray>();
```

Comment:
This works? 👀
Seems a bit cursed, but if it works, it works :D

Reply:
yup, dw about the weird arduino stuff lmao